Troubleshooting.md

```diff
@@ -70,17 +78,17 @@ Once the collection barcode duplication issue is resolved, the manifest ETL will
 Aborting with error: Identifier found in set «samples-haarvi», not «samples»
 ```
 This error means that a sample from a separate study arm that we're not supposed to be ingesting was not properly marked as experimental (`_exp`), so it ended up in our pipeline.
```

Comment:
Should this contain the same notes as above about being time-sensitive and blocking lab results? Anytime the presence-absence job is erroring and aborting it would have that impact.